Source/Scene/Points3DTileContent.js

```diff
@@ -166,11 +168,97 @@ define([
         var pointsLength = view.getUint32(byteOffset, true);
         byteOffset += sizeOfUint32;
 
-        var positionsOffsetInBytes = byteOffset;
-        var positions = new Float32Array(arrayBuffer, positionsOffsetInBytes, pointsLength * 3);
+        var batchTableJSONByteLength = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUint32;
+
+        var batchTableBinaryByteLength = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUint32;
+
+        var positions = new Float32Array(arrayBuffer, byteOffset, pointsLength * 3);
```

Comment:
Actually, in b3dm and i3dm, the batch table comes before glTF.  We should be consistent here for now and move the positions payload after the batch table; however, we might want to switch them all later if there is a benefit, e.g., to avoid parsing it in some cases?